pumps/hybrid.go

```diff
@@ -128,8 +130,25 @@ func (p *HybridPump) Init(config interface{}) error {
 			p.trackAllPaths = trackAllPaths.(bool)
 		}
 
-		if storeAnalyticPerMinute, ok := meta["store_analytics_per_minute"]; ok {
-			p.storeAnalyticPerMinute = storeAnalyticPerMinute.(bool)
+		if storeAnalyticsPerMinute, ok := meta["store_analytics_per_minute"].(bool); ok {
```

Comment:
I think we should consider here the use case in which `meta["store_analytics_per_minute"] `is nil, this will throw a panic when casting. Even if we think that this will never be nil we should validate it. Toughts?

Reply:
Do you mean first checking the value of "ok" and then, setting a type for storeAnalyticsPerMinute? something like this:
`	if storeAnalyticsPerMinute, ok := meta["store_analytics_per_minute"]; ok {
			p.storeAnalyticsPerMinute = storeAnalyticsPerMinute.(bool)
			p.aggregationTime = 1
		}`

Reply:
I think we are validating it through the "ok" variable, am I wrong? 
When store_analytics_per_minute is nil (it does not exist in the map), "ok" will be equal to false, and we're handling that situation. Let me know if there is something I am not realizing!